src/Rules/Playground/PhpdocCommentRule.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Playground;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Node\VirtualNode;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use function str_starts_with;
+
+/**
+ * @implements Rule<Node>
+ */
+final class PhpdocCommentRule implements Rule
+{
+
+	public function getNodeType(): string
+	{
+		return Node::class;
```

Comment:
I'd do it only for Node\Stmt.

Reply:
tbh there are so many different node types, that I don't know what the differences are in this case.

adjusted it per request :)